astroquery/mast/cutouts.py

```diff
@@ -106,8 +137,14 @@ def __init__(self):
                     }
         self._service_api_connection.set_service_params(services, "tesscut")
 
+    def _get_default_timeout(self):
+        """ Gets the default request timeout limit. """
+
+        return self._service_api_connection.TIMEOUT
```

Comment:
why is this method necessary? (I see in the history that there has been a bit of a back and forth, so maybe it's not needed any more?

Reply:
I included this for the case where a user might switch between `get_cutouts` and `download_cutouts` but does not anticipate that the `timeout` setting stayed the same. Having the `timeout` arg in each individual method implies that it would only change `timeout` for a call with that method and not any of the others, so I figured it would make sense to keep the changes contained by resetting that arg each time.

Reply:
The code has moved around since this last comment so I'm not sure what you're referring to, but I made a call to `_get_default_timeout` several lines below L414.

Reply:
But it doesn't provide the default timeout, but gets the attribute at all time. If you need the value from the attribute, do call it directly in the code (this is why I don't see the reason for this method), or if you really need the default, do reach out to it directly from the config.

Maybe it's just a naming issue? Though I still think the method is not necessary as it basically just calls one line that you could call directly when needed.